Sources/MongoSwift/SDAM.swift

```diff
@@ -24,11 +24,41 @@ public struct ServerAddress: Equatable {
 
     /// Initializes a ServerAddress, using the default localhost:27017 if a host/port is not provided.
     internal init(_ hostAndPort: String = "localhost:27017") throws {
```

Comment:
We'll also want the ability to parse unix domain sockets, but I think we should just punt that to a later PR if we have time at the end of the rotation (they're not super high priority). I've filed SWIFT-1174 to track the work for that.